.gitignore

```diff
@@ -7,6 +7,10 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # files that interfere with YARN
+package-lock.json
+# Allow package-lock.json in docs/
+!docs/package-lock.json
```

Comment:
This is not required. There is a correct entry for this in the docs/ directory. Please remove this line.